API.py

```diff
@@ -0,0 +1,87 @@
+import json
+import re
+import uuid
+from flask import Flask, render_template, request, jsonify, abort
+import os
+import time
+
+app = Flask(__name__)
+UPLOAD_FOLDER = 'uploads'
+OUTPUT_FOLDER = 'output'
+app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
+app.config['OUTPUT_FOLDER'] = OUTPUT_FOLDER
+the_files = {}
+
+
+@app.route('/')
+def home():
+    return render_template('home.html')
+
+
+@app.route('/upload', methods=['POST'])
+def upload():
+    if 'file' not in request.files:
+        return jsonify({'error': 'No file attached'}), 400
```

Comment:
Assign constants to descriptive variables to enhance readability and simplify future changes ^__^ . Directly using them can lead to confusion and maintenance challenges.